app/src/main/java/org/dhis2/utils/granularsync/SyncStatusDialog.kt

```diff
@@ -323,7 +321,10 @@ class SyncStatusDialog : BottomSheetDialogFragment(), GranularSyncContracts.View
                         syncSms()
                     }
                 } else {
+                    binding!!.connectionMessage
+                        .setText(R.string.sms_available_for_individual_records)
                     binding!!.syncButton.visibility = View.GONE
+                    binding!!.syncButton.setOnClickListener(null)
```

Comment:
Maybe now is a good time to remove the `!!` from the binding

Reply:
I think it is not a good idea to refactor a class so close from release